infra/docker/Dockerfile

```
@@ -46,12 +57,28 @@ COPY langwatch/package.json langwatch/pnpm-lock.yaml langwatch/pnpm-workspace.ya
 COPY langwatch/packages ./langwatch/packages
 COPY langwatch/vendor ./langwatch/vendor
 # https://stackoverflow.com/questions/70154568/pnpm-equivalent-command-for-npm-ci
-RUN cd langwatch && CI=true pnpm install --frozen-lockfile
+RUN CI=true pnpm install --frozen-lockfile --filter "@langwatch/web..."
 COPY langwatch ./langwatch
 RUN cd langwatch && NODE_OPTIONS=--max-old-space-size=4096 pnpm run build
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)